run_data_measurements.py

```diff
@@ -309,6 +310,10 @@ def main():
         server = smtplib.SMTP_SSL("smtp.gmail.com", port, context=context)
         server.login("data.measurements.tool@gmail.com", EMAIL_PASSWORD)
 
+    # The args specify that multiple features can be selected.
+    # We combine them for the filename here.
+    args.feature = ".".join(args.feature)
```

Comment:
Oh interesting, I thought that they were already being combined for the filename somewhere

Reply:
They had been, but in various code changes that ended up not working anymore. =/